packages/terra-dropdown-button/tests/wdio/dropdown-button-spec.js

```diff
@@ -263,13 +263,6 @@ Terra.describeViewports('Dropdown Button', ['medium'], () => {
       $('#first').click();
       Terra.validates.element('first button clicked');
     });
-
-    it('Focus to single item on arrow down', () => {
```

Comment:
same here !! I think this test was added validate item focus.

Reply:
Implemented as above.